ext/bg/js/dictionary.js

```diff
@@ -178,6 +318,26 @@ function dictTagsSort(tags) {
     });
 }
 
+function dictTermTagScore(tags) {
+    let score = 0;
+
+    const tagScores = {
```

Comment:
Maybe I'm missing something, but aren't these actually stored in the metadata in the dictionary JSON?

Reply:
The score for terms in `term_bank_n.json` is currently the result of `termTags` like `P` or `news` and `definitionTags` like `arch`. Maybe yomichan-import should be updated to have a score separately for both of these. But that would require changes to the sorting. `tag_bank_n.json` only has `[Name, Category, Order, Notes]`.

Reply:
In that case, I think it would be fine to introduce `Score` to the tag array (at the end, as usual). As you've probably noticed, a lot of effort is made to make sure that all data comes from data files only. The only exception is for styling (that's why it is OK that tag colors are in the style sheets).